command/base64/base64.go

```diff
@@ -0,0 +1,136 @@
+package base64
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/smallstep/cli/command"
+	"github.com/smallstep/cli/utils"
+	"github.com/urfave/cli"
+)
+
+func init() {
+	cmd := cli.Command{
+		Name:      "base64",
+		Action:    command.ActionFunc(base64Action),
+		Usage:     "encodes and decodes b64 data.",
```

Comment:
I don't think we normally have a period at the end of the usage

Reply:
Fixed with 592fd53